src/cache-manager.ts

```diff
@@ -161,10 +162,18 @@ export class CacheManager {
     // ** Image **
     else if (
       isFileImage(path) &&
-      this.plugin.settings.imagesIndexing &&
-      extractor?.canFileBeExtracted(path)
+      ((this.plugin.settings.imagesIndexing &&
+      extractor?.canFileBeExtracted(path)) ||
+      (this.plugin.settings.aiImageIndexing &&
+      aiImageAnalyzer?.canBeAnalyzed(file)))
     ) {
-      content = await extractor.extractText(file)
+      if (this.plugin.settings.imagesIndexing && extractor?.canFileBeExtracted(path)){
+        content = content + await extractor.extractText(file)
```

Comment:
`content` is (and should be) null at this point, no need to append it

Reply:
I removed it✅